### ISSUE TITLE
Axe add `color-contrast-enhanced`

### ISSUE THREAD

**Issue by adamraine** [GoogleChrome/lighthouse#13815]
https://github.com/GoogleChrome/lighthouse/pull/13814#issuecomment-1087919687

New axe rule added in 4.4.0.

**Comment by connorjclark** [comment]
There's a lot of current discussion about how the color contrast rule is lacking: https://github.com/w3c/wcag/issues/695 / https://blog.datawrapper.de/color-contrast-check-data-vis-wcag-apca/

This enhanced rule is a lot stricter, and will probably be a thorn more than it'd be helpful. We'll defer adding it, hopefully the a11y standards are updated with better contrast guidance soon.

**Comment by Myndex** [comment]
Just as a note:

color-contrast-enhanced is (normally) only a WCAG 2 thing, specifically 1.4.6, and there is no relation to that and the APCA, the candidate method for WCAG 3. I mention this as there are links above that cite APCA.

1.4.6, and all contrast SCs of WCAG 2 are using the same flawed math, which is not perceptually accurate.

Please let me know of any questions.

Thank you!
